auctions/templates/includes/payout.html

```diff
@@ -15,8 +15,7 @@
        {% else %}
 
            <h3>You need to tell us where to send your payout.</h3>
-
-           <li><a class="success button expanded" href="{% url 'bank' %}">Register a bank account</a></li>
+           <li><a class="success button expanded" href="{% url 'bank' %}?return_url={{ return_url }}">Register a bank account</a></li>
```

Comment:
Not introduced in this PR, but the `<li>` element here still has a bullet style.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/71928/27498167-6e62672c-5823-11e7-859d-c5d8f4f9f0f3.png">